accounting/report.go

```diff
@@ -0,0 +1,47 @@
+package accounting
+
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// Report contains a set of entries.
+type Report []*HarmonyEntry
+
+// HarmonyEntry represents a single action on our balance.
+type HarmonyEntry struct {
+	// Timestamp is the time at which the event occurred.
+	// On chain events: timestamp will be obtained from the block timestamp.
+	// Off chain events: timestamp will be obtained from lnd's records.
+	Timestamp time.Time
+
+	// Amount is the balance change incurred by this entry, expressed in
+	// msat.
+	Amount lnwire.MilliSatoshi
+
+	// Fiat is the fiat value of this entry's amount.
+	Fiat float64
```

Comment:
nit: Maybe calling this `FiatValue` could work even better.
Q: shouldn't we indicate the currency type as well?

Reply:
Fiat conversion is handled by another package, which _should_ have multiple fiat conversions supported. To get these reports, you'd query with a currency param. 

But `FiatValue` sgtm!

Reply:
Fiat conversion is handled by another package, which _should_ have multiple fiat conversions supported. To get these reports, you'd query with a currency param. 

But `FiatValue` sgtm!